pgbouncer-exporter/version.txt

```diff
@@ -1 +1 @@
-0.15.0-2
+0.16.0
```

Comment:
why pgbouncer version has been updated it should be 0.15.0-3

Reply:
good catch!

Reply:
@pgvishnuram i refreshed your PR with changes that will correct this mistake https://github.com/astronomer/ap-vendor/pull/619